ui/revenuecatui/gradle.properties

```diff
@@ -0,0 +1,4 @@
+POM_NAME=revenuecat-ui
+# We default to deploy the noop version since release is the noop version.
+POM_ARTIFACT_ID=revenuecat-ui
```

Comment:
I prefer this name than the one we currently use which is `com.revenuecat.purchases:revenuecatui` since it's more in line with our amazon and debug view dependencies. I think we should get this merged before having people test the library with new apps at least so they use the correct dependency name, cc @NachoSoto

Reply:
The debug view one is `purchases-debug-view` and the amazon one is `purchases-amazon`. Do we need the `revenuecat` part? In iOS the regular SDK is called RevenueCat, but we are still calling the android one `purchases-android`

I wonder if just `ui` works too since it's part of `com.revenuecat.purchases:`

Reply:
Hmm honestly I was between `revenuecat-ui` and `purchases-ui`. While we are under the `com.revenuecat.purchases` group, I think we need a bit more detail in case we ever have multiple ui libraries (or even right now with debugview and the paywall). Do you think `purchases-ui` might be better @vegaro ?